package.json

```diff
@@ -39,6 +39,7 @@
     "coveralls": "^3.0.0",
     "detect-node": "^2.0.3",
     "documentation": "^5.2.2",
+    "electron": "^1.7.9",
```

Comment:
This is a pretty big download, it would be nice to make this optional and only install when needed